JASP-Desktop/engine/enginerepresentation.cpp

```diff
@@ -113,7 +113,6 @@ void EngineRepresentation::clearAnalysisInProgress()
 
 void EngineRepresentation::setAnalysisInProgress(Analysis* analysis)
 {
-	analysis->incrementRevision(); // Increment revision only when the analysis request is about to be sent
```

Comment:
I remember this fixed a nasty bug a while ago @boutinb (https://github.com/jasp-stats/jasp-desktop/commit/82195d175182488587fe4afdfdc0a763eb38d593). I can't remember the exact context though. Do you? Something about the analysis having a different revision status then what was returned from the analysis? (You need to add more information to your pull requests ;))

Reply:
Yes I have to have a look. Next week.

Reply:
It would be so practical if there was more info in @boutinb commits and PRs ;)

Reply:
Like a link to an issue or something.